Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/sparkserverlessnode/SparkServerlessClusterNode.java

```diff
@@ -67,7 +67,7 @@ public SparkServerlessClusterNode(@NotNull Node parent,
     protected void refreshItems() throws AzureCmdException {
         try {
             cluster.get().toBlocking().singleOrDefault(cluster);
-            if (Optional.ofNullable(cluster.getMasterState()).orElse("")
+            if (getNodeActionByName("Update") == null && Optional.ofNullable(cluster.getMasterState()).orElse("")
```

Comment:
I suppose you may need to remove all actions before adding one. The method name is `refreshItems`